src/services/eventService.ts

```diff
@@ -1,41 +1,47 @@
-import { v4 as uuidv4 } from 'uuid';
 import md5 from 'md5';
-import MeanApiService from './meanApiService';
+import MixpanelLibray, { Mixpanel, Response } from 'mixpanel-browser';
 import ProviderService from './providerService';
+import { NETWORKS } from '@constants/addresses';
+import find from 'lodash/find';
 
 export default class EventService {
-  meanApiService: MeanApiService;
-
   providerService: ProviderService;
 
-  sessionId: string;
-
-  identifierId: string;
+  mixpanel: Mixpanel;
 
-  constructor(meanApiService: MeanApiService, providerService: ProviderService) {
-    this.meanApiService = meanApiService;
+  constructor(providerService: ProviderService) {
     this.providerService = providerService;
-    this.sessionId = uuidv4();
+    this.mixpanel = MixpanelLibray.init(
+      '7f62e8855fcbcd72de460ba40b6fc06b',
+      { api_host: 'https://mean-finance-mixpanel-proxy.herokuapp.com' },
+      ' '
+    );
+    this.mixpanel.set_config({ persistence: 'localStorage', ignore_dnt: true });
   }
 
   async getIdentifier() {
     const account = await this.providerService.getAddress();
-
-    return (account && md5(account)) || md5(this.sessionId);
+    return md5(account);
```

Comment:
account can be undefined before user connects its wallet, we should maintain the sessionId to at least track users that don't connect their wallet, we could send both so we also track the session

Reply:
AFAIK mixpanel already handles this without the need of a `session_id`.  Identifier should be per/user - but sadly mixpanel only allows one user identified per "device" (I can clarify this on a meet).

Reply:
Just found out [about this](https://docs.mixpanel.com/docs/analysis/advanced/sessions), we can create a `session_id` but we should def. not use the wallet address for that, just the uuidv4 (will update code).

Reply:
sure, but do you not want to track specific wallets?

Reply:
Nain